app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,8 @@
+module MailerHelper
+  def recipient_name
```

Comment:
I don't think this justifies a separate helper, but if you think it'll be worth it in the future feel free to leave it in.